src/cdk/table/table.spec.ts

```diff
@@ -825,6 +825,26 @@ describe('CdkTable', () => {
         }).not.toThrow();
       });
 
+      it('should be able to detect row template changes and re-render', () => {
+        setupTableTestApp(WhenRowChangeDetectionCdkTableApp);
+        fixture.detectChanges();
+
+        let dataRows: Element[] = getRows(tableElement);
+        dataRows.forEach( element => {
```

Comment:
```suggestion
        dataRows.forEach(element => {
```
extra space (here and below)